deployments/dev/environment.py

```diff
@@ -40,7 +40,7 @@ def env() -> Mapping[str, Optional[str]]:
             'hca:it2ebi:repository/tdr:metadata/hca'
         ]),
 
-        'AZUL_TDR_SOURCES': ','.join([
+        'AZUL_TDR_SOURCE': ','.join([
```

Comment:
Was this change intentional? It seems unrelated and breaks the build

Reply:
Yeah its in a drop commit to show how/why things are failing, this failure shows that the failure is now contained to a single test case, and does not spread to other tests.

Reply:
I'm still not clear on what this commit is supposed to demonstrate. You changed the name of an environment variable and [observed](https://travis-ci.com/github/DataBiosphere/azul/jobs/482481471) the obvious consequence, a `KeyError` when the configuration tries to read that variable. I don't see any other errors. How then, can this failure be considered "collateral"?

If you're just trying to show that the environment mis-configuration _doesn't_ cascade into other test failures, then the commit title is wrong.

Reply:
@amarjandu, this is a very roundabout way of causing the tests to fail. I can understand Noah's confusion.

Why don't you just raise an explicit exception in the test or tests that you want to show cause collateral failures of other tests?